cmd/server/server.go

```diff
@@ -0,0 +1,153 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"time"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/freeverseio/laos-universal-node/internal/platform/blockchain"
+	internalRpc "github.com/freeverseio/laos-universal-node/internal/rpc"
+)
+
+type HTTPServerer interface {
```

Comment:
I know, naming is hard. I am not blaming you, it's just that I don't like `ServerER`. What if we rename the interfaces `HTTPServer` and `RPCServer` and the implementing structs `HTTP` and `RPC`? I'm open to proposals.